docs/rest.md

```diff
@@ -339,6 +339,43 @@ const link = new RestLink({
 
 > When using the object form, the `uri` field is required.
 
+<h3 id=options.example.customFetch>Custom Fetch</h3>
+
+By default, Apollo uses the browsers `fetch` method to handle `REST` requests to your domain/endpoint. The `customFetch` option allows you to specify _your own_ request handler by defining a function that returns a `Promise` with a fetch-response-like object:
+
+<h3 id=options.example.customFetch>Custom Fetch</h3>
```

Comment:
There's some duplicated text here, lines 342-348 has two times the section header and paragraph.

Reply:
Just fixed that stupid copy past error...